src/Components/NavBar/NavBar.tsx

```diff
@@ -1,11 +1,24 @@
 import * as React from 'react';
 import './NavBar.scss';
+import writingLogo from './WritingLogo.png';
+import { NavLink } from 'react-router-dom';
 
 const NavBar: React.FC = () => {
   return (
-    <nav>
+    <header>
+      <img className='logo' src={writingLogo} alt='One Minute Writer'/>
       <h1>One Minute Writer</h1>
-    </nav>
+      <nav>
+        <ul className='menu'>
+            <li>
+              <NavLink to='/' className='selected' id="dashboard-nav-button">Dashboard</NavLink>
```

Comment:
NavLinks are good to use here. This will help us underline the correct button on the navbar.